src/operations/query/filters/dateTime.js

```diff
@@ -5,6 +5,83 @@ function isPeriodField(fieldString) {
     return fieldString === 'period' || fieldString === 'effectivePeriod';
 }
 
+/**
+ * filters by date for a Period
+ * https://www.hl7.org/fhir/search.html#date
+ * https://www.hl7.org/fhir/search.html#prefix
+ * @param {string} dateQueryItem
+ * @param {string} fieldName
+ * @returns {Object[]}
+ */
+function datetimePeriodQueryBuilder({ dateQueryItem, fieldName }) {
```

Comment:
All query builders are in the file `querybuilder.util.js`. We should move this as well.